src/Adapter/Cart/Comparator/CartProductUpdate.php

```diff
@@ -56,13 +62,15 @@ class CartProductUpdate
     /**
      * @param int $productId
      * @param int $combinationId
+     * @param int $customizationId
      * @param int $deltaQuantity
      * @param bool $created
      */
-    public function __construct(int $productId, int $combinationId, int $deltaQuantity, bool $created)
+    public function __construct(int $productId, int $combinationId, int $customizationId, int $deltaQuantity, bool $created)
```

Comment:
Bad idea to put it at the middle. Why not put it at the end and set it as optional with null as default value ?

Reply:
I thought about it but as this class is new (introduced in 1.7.7.0), I thought it was cleaner to put this parameter just after `$combinationId`.

Reply:
The problem is the code is already in production. So we must think to users. 

Could you move the parameter ?

Reply:
The parameter must be after, otherwise, it's a bc break. If the parameter is mandatory, we can't accept it, but it's still a bc breaks :sweat_smile: